### PR TITLE
OSDOCS-11956: Multi-line Code Copy/Paste Incorrect

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-11956](https://issues.redhat.com//browse/OSDOCS-11956): Multi-line Code Copy/Paste Incorrect

Version(s):
4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-11956

Link to docs preview:
https://81582--ocpdocs-pr.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-aws-secret-manager.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @paulczar - When you have a moment, can you please review this PR? Thanks!

File: cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc [hunk 1]
```diff
@@ -142,8 +142,7 @@ $ oc label csidriver.storage.k8s.io/secrets-store.csi.k8s.io security.openshift.
 $ SECRET_ARN=$(aws --region "$REGION" secretsmanager create-secret \
     --name MySecret --secret-string \
     '{"username":"shadowman", "password":"hunter2"}' \
-    --query ARN --output text)
-$ echo $SECRET_ARN
+    --query ARN --output text); $ echo $SECRET_ARN
```
Comment (paulczar) [inline cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc:145]: should be `--query ARN --output text);  echo $SECRET_ARN`

**Review by paulczar — APPROVED** [review]

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**